fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/Loan.java

```diff
@@ -6086,7 +6087,11 @@ public void updateLoanScheduleOnForeclosure(final Collection<LoanRepaymentSchedu
         }
     }
 
-    private boolean isForeclosure() {
+    public Integer getLoanSubStatus() {
```

Comment:
Lombok getter is on the Loan entity, so probably this getter method can be removed.